account_consolidation/models/account.py

```diff
@@ -23,7 +23,8 @@ def _compute_conso_company(self):
     consolidation_account_id = fields.Many2one(
         comodel_name='account.account',
         string='Consolidation account',
-        domain="[('company_id', '=', consolidation_company_id)]",
+        domain="[('company_id', '=', consolidation_company_id),"
+               "('user_type_id', '=', user_type_id)]",
```

Comment:
:warning: We can't force the consolidation on the same type of account.

Reply:
this was made to avoid issue which we have in consolidation,without domain user will that any type of account is accessible and it' not,  or I can add a help message, at least something

Reply:
IMO those who want to use this module must know what consolidation is about. 
So if the balance of all the accounts mapped to the same account is 0, it's normal that no consolidation entries are generated.
Maybe you can add a note in the known issues that states "Functional mapping of accounts is not tested. If no consolidation entries are generated, it's probably because the balance of mapped accounts is 0."